tests/test_plugins.py

```diff
@@ -52,7 +52,6 @@ def test_plugin_structure(self):
             language=self.language,
             picture=self.picture,
         )
-        self.page.publish(self.language)
         self.assertEqual(plugin.get_plugin_class_instance().name, "Image")
```

Comment:
```suggestion
        if cms_version < "4":
            self.page.publish(self.language)
        self.assertEqual(plugin.get_plugin_class_instance().name, "Image")
```
Might do the trick.

Reply:
Is there a reason the page is being published again?